src/python/WMCore/WorkQueue/WorkQueueBackend.py

```diff
@@ -171,10 +170,11 @@ def insertElements(self, units, parent=None):
             if unit._couch.documentExists(unit.id):
                 self.logger.info('Element "%s" already exists, skip insertion.' % unit.id)
                 continue
-            else:
-                newUnitsInserted.append(unit)
+
+            newUnitsInserted.append(unit)
             unit.save()
-            unit._couch.commit(all_or_nothing=True)
+            # FIXME: this is not performing bulk request, but single document commits(!)
```

Comment:
Why is it needed to change the behavior here, from `all_or_nothing=True` to single document uploads?

Reply:
Ok, let me answer  myself, you've already explained it all in the issue comments.
> CouchDB all_or_nothing option is no longer available for bulk APIs

Sorry for the noise! :)